release v1.32.0-rc2

Stebalien opened the pull request:
Builds with go 1.24

rvagg reviewed (approved):
ok, but what's going on with this? https://github.com/filecoin-project/filecoin-ffi/releases/tag/v1.32.0-rc2, and why was I just notified about it and why does it say "2 weeks ago"?

Stebalien commented:
> ok, but what's going on with this? https://github.com/filecoin-project/filecoin-ffi/releases/tag/v1.32.0-rc2, and why was I just notified about it and why does it say "2 weeks ago"?


Crap, I deleted that release and thought I had already deleted that tag. I hope this works...

Stebalien commented:
Basically, the old release process was "tag a release, things will happen". The new release process is "update version.json". I did it the old way and it obviously didn't actually build a release.
